src/pages/en/core-concepts/framework-components.md

```diff
@@ -121,9 +120,11 @@ import MyButton from '../components/MyButton.svelte';
 </MySidebar>
 ```
 
-Framework components can only contain other components of the same framework. For example, a single React component can have an entire tree of React child components, but cannot contain Astro components or Vue components. Only Astro components can contain child components from any framework.
+⚠️ *Note that framework components themselves (e.g. `.jsx`, `.svelte`) cannot mix multiple frameworks.*
+
+This allows you to build entire "apps" in your preferred JavaScript framework and render them, via a parent component, to an Astro page. This is a convenient pattern to allow related components to share state or context.
 
-This allows you to build entire "apps" in your preferred JavaScript framework and render them, via a parent component, to an Astro page. This is a convenient pattern to allow related components to share state or context. 
+Each framework has its own patterns for nesting: `children` props and _render props_ for React and Solid; `<slot>` with or without names for Svelte and Vue, for example. Note, however, that when you include a framework component in an astro component, you can't use _render props_ or _named slots_, even if the framework component supports it. This is due to a limitation in Astro's compiler.
```

Comment:
Each framework has its own patterns for nesting: `children props` and render props for React and Solid; `<slot>` with or without names for Svelte and Vue, for example. Note, however, that when you include a framework component in an astro component, you can't use render props or named slots, even if the framework component supports it. This is due to a limitation in Astro's compiler.